ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/patch-panel.j2

```diff
@@ -7,6 +7,27 @@
 {% if patch_panel is arista.avd.defined %}
 !
 patch panel
+{%     if patch_panel.connector is arista.avd.defined %}
+{%         set pp_connector_cli = "connector " %}
+{%         if patch_panel.connector.interface is arista.avd.defined %}
+{%             set pp_connector_cli = pp_connector_cli ~ "interface " %}
```

Comment:
You can collapse tests, since testing for deeper keys always includes the check for parents.

Also I don't think we need a cli variable here, since you are not really adding multiple tokens "building" a long command.
```suggestion
{%     if patch_panel.connector.interface is arista.avd.defined %}
```

Reply:
the idea was to prepare for possible other commands in the future, but i agree to slim it down for now